docs/fundamentals/flow/create-flow.md

```diff
@@ -425,6 +425,8 @@ This will get you the following output:
 
 So both `BarExecutor` and `BazExecutor` received only received a single `Document` from `FooExecutor` as they are run in parallel. The last Executor `executor3` will receive both DocumentArrays and merges them automatically.
 
+The automated merging can be disabled by setting `disable_reduce=True`. This can be useful when you need to provide your custom merge logic in a separate Executor. In this case the last `.add()` call would like `.add(needs=['barExecutor', 'bazExecutor'], uses=CustomMergeExecutor, disable_reduce=True)`
```

Comment:
Is this approach sustainable? It would mean that we will end up with references to versions all over the docs? I think the docs will only be deployed with the next release, no? Without versioning the docs it means the docs are always only valid for the latest release?

Reply:
Well, at least for a while I prefer to have this. I agree is not sustainable long-term